astropy/wcs/wcsapi/fitswcs.py

```diff
@@ -118,6 +118,7 @@
     'AWAV': 'em.wl',  # Air wavelength
     'VELO': 'spect.dopplerVeloc',  # Apparent radial velocity
     'BETA': 'custom:spect.doplerVeloc.beta',  # Beta factor (v/c)
+    "STOKES": "phys.polarization.stokes", # STOKES parameters
```

Comment:
```suggestion
    'STOKES': 'phys.polarization.stokes',  # STOKES parameters
```
Single quotes just for consistent looks; double space for E261